### DISCUSSION TITLE
希望支持点击时间快速空降(评论或弹幕里的时间点)

### DISCUSSION THREAD

**Discussion by Yumeyo23** [ywmoyue/biliuwp-lite#622] [feature]
### 问题描述

希望支持快速跳转，点击别人发送的时间可以跳转
![PixPin_2024-05-09_00-00-34](https://github.com/ywmoyue/biliuwp-lite/assets/88522326/c055724e-89d6-4ac1-b264-78edfe6537a5)

### 描述解决方案

如上图所示，官网B站支持

### 附加信息

稀饭作者~每次我的反馈作者大大都会看，真的太赞啦，有什么交流群之类的吗，想一起玩儿蛤蛤蛤，作者辛苦啦

### 代码贡献

否

**Comment by Yumeyo23** [comment]
会有吗awa

**Reply by ywmoyue** [reply]
后面看看

**Comment by Shaddoll126** [comment]
非常实用的功能  希望能加上
